app/controllers/facilities_controller.rb

```diff
@@ -12,4 +12,76 @@ def validate_params
   rescue ArgumentError
     raise Common::Exceptions::InvalidFieldValue.new('bbox', params[:bbox])
   end
+
+  private
+
```

Comment:
Not sure that this code belongs in the controller, could you implement a helper/module instead for this to keep the controller clean?

Reply:
I could do that. when this gets reused and we are killing off active model serializer this should move into application_controller.rb
https://medium.com/@anMagpie/rails-api-with-fastjson-from-netflix-e7bf4f4e5e7b